src/olympia/reviewers/forms.py

```diff
@@ -114,6 +115,28 @@ class VersionsChoiceWidget(forms.SelectMultiple):
         amo.STATUS_AWAITING_REVIEW: ('reject_multiple_versions',),
     }
 
+    def get_extended_status(self, obj):
+        if not obj or not obj.file:
+            return
+        # This is the default status
+        status = obj.file.get_status_display()
```

Comment:
I assume this will show "Disabled by Mozilla" for rejected files? If I understand and remember correctly, we also (wrongfully) use that label when the developer disables a version. Can we distinguish the two here?

If so, let's use "Rejected" and "Disabled by Developer".
If not, we'll probably need go to with something like "Rejected or Disabled by Developer".

Reply:
afaik we can reliably know if a version has been disabled by the developer (`version.is_user_disabled`) but differentiating normal rejection from being superseded by a later upload (or manually changed in the admin) is tricky.  We attempt it in elsewhere on the review page - if there's a review date we say "Rejected", otherwise "Rejected or Unreviewed" - so we could do the same here.

Reply:
tbh it's annoying this extra status information logic is buried in obscure reviewer tools jina helper functions!

Reply:
I see. It might be an easier approach then to say "Disabled by Developer" if that happened and "Disabled by Mozilla" otherwise. Thoughts?

Reply:
I was going to reuse the same code we use for display file status on the review page for consistency (but refactor it out of jinja_helpers)

Reply:
Ah hm. My concern is that the file status display on the review page might not be accurate. It's possible it might be missing `is_user_disabled`. It sounded logical to me not to repeat (or reuse) this bug, but we can also fix in a followup if you prefer.

Reply:
(this will be in another commit - WIP)

Reply:
https://github.com/mozilla/addons-server/pull/19465/commits/c4e2ceb03bf09abdf5fa8fafa9f6847503c23b47 adds these status functions, and moves the logic out of forms.py too so it's all in one place, in the model.